src/app/api/routes/alerts.py

```diff
@@ -70,6 +70,16 @@ async def update_alert(payload: AlertIn, alert_id: int = Path(..., gt=0)):
     return await crud.update_entry(alerts, payload, alert_id)
 
 
+@router.put("/{alert_id}/acknowledge", response_model=AlertOut, summary="Acknowledge an existing alert")
+async def acknowledge_alert(alert_id: int = Path(..., gt=0)):
+    """
+    Based on a alert_id, acknowledge the specified alert
+    """
+    payload = await crud.get_entry(alerts, alert_id)
+    payload['is_acknowledged'] = True
+    return await crud.update_entry(alerts, AlertIn(**payload), alert_id)
```

Comment:
I would suggest to replace by:
```suggestion
from schemas import Acknowledgement

async def acknowledge_alert(alert_id: int = Path(..., gt=0)):   
    return await crud.update_entry(alerts, Acknowledgement(is_acknowledged=True), alert_id)
```
That way if the alert_id doesn't exist you get the usual exception returned by update_entry and you avoid having to do two calls.

Reply:
Done!